api/DragEvent.json

```
@@ -20,7 +20,8 @@
             "version_added": "4"
           },
           "ie": {
-            "version_added": "10"
+            "version_added": "9",
+            "notes": "`DragEvent` has been implemented natively by Internet Explorer 9 under its standard mode but not quirks mode."
```

A suggestion to make it a little more explicit what's going on here and to follow our typical notes style:

```suggestion
            "notes": "Before Internet Explorer 10, <code>DragEvent</code> is exposed in standards mode but not quirks mode."
```